js/main.js

```diff
@@ -1 +1,24 @@
-// Your functions or code that should load on all pages goes here.
+function onVersionChange(form, animation) {
+    if (typeof version_prefixes == 'undefined' || typeof fields_per_version == 'undefined'
+        || !version_prefixes || !fields_per_version) {
```

Comment:
If for some reason the 2 variables we passed from python are not defined or are empty, we don't do anything (and all fields will remain shown).